src/routes/relay/relay.controller.ts

```diff
@@ -8,7 +8,10 @@ import { RelayDtoValidationPipe } from '@/routes/relay/pipes/relay.validation.pi
 @ApiTags('relay')
 @Controller({
   version: '1',
-  path: 'chains/:chainId/relay',
+  path: [
+    'relay/:chainId', // TODO: Remove after client has migrated
+    'chains/:chainId/relay',
+  ],
```

Comment:
Having a different controller for the legacy route can be more beneficial to separate concerns (legacy vs non-legacy routes). We can accomplish this with a `@Redirect` which would return a 302 by default to the clients signalling that the resource was moved which is also what we want to signal.


```typescript
@Controller({
  version: '1',
  path: 'relay/:chainId',
})
export class RelayLegacyController {
  @Post()
  @Redirect('/v1/chains/:chainId/relay')
  relay(): void {}

  @Get(':safeAddress')
  @Redirect('/v1/chains/:chainId/relay/:safeAddress')
  getRelaysRemaining(): void {
  }
}
```

Reply:
Changed in 4a94684980e5829ef6d81cf00a298c4d22ee7027.